src/cotonic.model.window.js

```diff
@@ -53,10 +52,10 @@ var cotonic = cotonic || {};
                 status:0,           // whether a status line appears at the bottom of the window {1 (YES) or 0 (NO)}.
                 width:500,          // sets the width in pixels of the window.
                 name:null,          // name of window
-                location:null,      // url used for the popup
```

Comment:
This wasn't used?

Reply:
It was defined double.

Reply:
O wait, this is not going to work. It was defined as a 0, 1 option to show the url, but later it is used as the href of the window.